apps/src/studio/studio.js

```diff
@@ -618,7 +618,7 @@ var calcMoveDistanceFromQueues = function (index, modifyQueues) {
   Studio.eventHandlers.forEach(function (handler) {
     var cmd = handler.cmdQueue[0];
     if (cmd && cmd.name === 'moveDistance' && cmd.opts.spriteIndex === index) {
-      var distThisMove = Math.min(cmd.opts.queuedDistance,
+      var distThisMove = Math.min(cmd.opts.queuedDistance || Infinity,
```

Comment:
I don't fully understand why this is needed, but `cmd.opts.queuedDistance` is occasionally undefined and `Math.min(undefined, 2)` is `NaN`.